mock/docs/site-defaults.cfg

```diff
@@ -148,7 +148,7 @@
 # case, you may need to have "bootstrap" chroot pre-populated from a container
 # image first (where the package manager stack is already pre-installed, so
 # mock doesn't have to).
-#config_opts['use_bootstrap_image'] = False
+#config_opts['use_bootstrap_image'] = True
```

Comment:
This file documents the defaults. And the default is False.

Reply:
reasonable